portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,211 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Yichen Yao</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <!-- Header -->
+    <div class = "header"> Yichen Yao</div>
+    <hr align="center" width="50%">
+
+    <!-- Snap Scroll Start -->
+    <article class="scroller">
+      <!-- Introduction -->
+      <section>
+        <div class="row">
+            <div class="column">
+                <img src="images/me.jpeg"  width="250" height="250">
+            </div>
+
+            <div class="column">
+                <div class="about">Hi, I'm Yichen.</div>   
+                <br>
+                <div class="about">Originally from Urbana, IL, I am a rising junior at <a href="https://www.columbia.edu/">Columbia University</a> studying computer science. </div>
+                <br>
+                <div class="about">In my free time, I love to go to concerts, check out new restaurants, and run.</div>
+                <br>
+                <br>
+              <div class="about">Currently reading <a href="https://www.goodreads.com/book/show/52163147-the-hidden-girl-and-other-stories">The Hidden Girl</a>, watching <a href="https://www.imdb.com/title/tt5788792/">The Marvelous Mrs Maisel</a>, learning to play tennis.</div>
```

Comment:
The line is pretty long, is there any way to solve it?

I am wondering if you are using any kind of html/css auto formatter? Since Hailun commented on the indentation, I guess a formatting tool could solve both of the issues.

Reply:
done.